packages/dds/merge-tree/src/test/client.conflictFarm.spec.ts

```diff
@@ -75,17 +76,29 @@ function runConflictFarmTests(opts: IConflictFarmConfig, extraSeed?: number): vo
 				config: { ...opts, applyOpDuringGeneration: true },
 			},
 			{
-				name: "with obliterate",
+				name: "obliterate with number endpoints",
 				config: {
 					...opts,
 					operations: [...opts.operations, obliterateRange],
 				},
 			},
+			{
+				name: "obliterate with sided endpoints",
+				config: {
+					...opts,
+					operations: [...opts.operations, obliterateRange, obliterateRangeSided],
+				},
+			},
```

Comment:
it looks like this is the only test failing, so lets comment it out, and check this in. this will make debugging easier for everyone, as they just need to un-comment to run it.